### PR TITLE
Add doc of one-line quick start script for different storage types

### PR DESCRIPTION
**Pull request by kezhenxu94**

### CODE REVIEW

**Review by wu-sheng — DISMISSED** [review]

**Comment by wu-sheng** [comment]
We could prepare the demo/showcase replacement, but it should be updated when BanyanDB 0.6 is ready.

File: docs/en/setup/backend/backend-docker.md [hunk 1]
```diff
@@ -1,5 +1,24 @@
...
+bash <(curl -sSL https://skywalking.apache.org/quickstart-docker.sh) 
+```
+
+You will be prompted to choose the storage type, and then the script will start
+the backend cluster with the selected storage.
+
+To tear down the cluster, run the following command:
+
+```shell
+docker compose --project-name=skywalking-quickstart down
```
[the remark this answers is not in the document]
Comment (kezhenxu94) [reply docs/en/setup/backend/backend-docker.md:19]: At the end of the script, the following log will be printed so the users should be very clear which port to use, I can add BanyanDB UI port there if users choose BanyanDB as storage

```
# ...
SkyWalking is now running. You can send telemetry data at localhost:11800 and access the UI at http://localhost:8080
```

[the remark this answers is not in the document]
Comment (wu-sheng) [reply docs/en/setup/backend/backend-docker.md:19]: This is good, we should expose BanyanDB UI as well, to query the data from BanyanDB from Browser.

**Comment by wu-sheng** [comment]
Is the exposed port visible in the logs too?

**Comment by kezhenxu94** [comment]
> Is the exposed port visible in the logs too?

Added in the website repo https://github.com/apache/skywalking-website/commit/f5532574b33b43f4d11aabc6944db97ea9761ec6

**Review by wu-sheng — APPROVED** [review]

**Pull request merged**